sqlite-db/src/closed.rs

```diff
@@ -129,7 +129,8 @@ impl Connection {
                 fees as "fees: models::Fees",
                 expiry_timestamp,
                 lock_txid as "lock_txid: models::Txid",
-                lock_dlc_vout as "lock_dlc_vout: models::Vout"
+                lock_dlc_vout as "lock_dlc_vout: models::Vout",
+                trading_pair as "trading_pair: models::TradingPair"
```

Comment:
🛠 Something it odd here! we are loading it but not inserting it into `closed_cfds` - same for `failed_cfds`!

Reply:
added! I noted that 2 other parameters from the Cfd are missing - was that on purpose???